Neos.ContentGraph.DoctrineDbalAdapter/src/Domain/Projection/Feature/SubtreeTagging.php

```diff
@@ -196,6 +196,8 @@ private function moveSubtreeTags(ContentStreamId $contentStreamId, NodeAggregate
               AND h.dimensionspacepointhash = :dimensionSpacePointHash
         SQL;
         try {
+            // Mysql hack
```

Comment:
a few more details about the why and how of that hack might be helpful for our future self…